aten/src/ATen/mps/MPSDevice.h

```diff
@@ -26,8 +26,7 @@ namespace at::mps {
 
 // Helper enum to check if a MPSGraph op is supported in a given macOS version
 enum class MacOSVersion : uint32_t {
-  MACOS_VER_13_0_PLUS = 0,
```

Comment:
Are we removing support for 13.0 as well? I think stable one moves to 13.1 , so should be fine, but we don't need to remove it as part of this PR ?

Reply:
No, 13.0 is technically supported, but there are no need for enum value for it (same as we did not have an enum for 12.3, because it was supported by default)